packages/components/Text.tsx

```diff
@@ -0,0 +1,15 @@
+import type { ReactNode } from "react";
+
+interface TextProps {
+  children?: ReactNode;
+  as?: "p" | "span";
+  style?: string;
+}
+
+export function Text({
+  children,
+  as: Tag = "p",
+  style = "",
+}: Readonly<TextProps>) {
+  return <Tag className={style}>{children}</Tag>;
+}
```

Comment:
Using the `style` props is a antipattern.

- React, by default, uses the `style` props for inserting inline styles by passing css
- React uses `className` prop to receive classes
- if we change this, we'll confuse users

**Suggestion**: change `style` to `className`, at the TextProps

References:
- https://legacy.reactjs.org/docs/dom-elements.html#style
- https://legacy.reactjs.org/docs/faq-styling.html

Reply:
Made in: https://github.com/podcodar/webapp/pull/173/commits/bdb394dc4841c83969c37e18cb0db7e8b8a19dac